docs/ingress/kong/sni-ingress/README.md

```diff
@@ -14,7 +14,7 @@ When leveraging a single endpoint ingress / load balancer we lose the ability to
     k3d load image --cluster k3s-default \
       datastax/cass-operator:1.3.0 \
       datastax/cass-config-builder:1.0-ubi7 \
-      datastax/dse-server:6.8.4-ubi7 \
+      datastax/dse-mgmtapi-6_8:6.8.4-ubi7 \
```

Comment:
This image does not exists.

Reply:
I updated this so we're talking about a recent version. The UBI will exist tomorrow :)